sdk/tests/deqp/functional/gles3/es3fShaderPrecisionTests.js

```diff
@@ -285,10 +285,9 @@ goog.scope(function() {
 
 		bufferedLogToConsole("Assuming " + mantissaBits + " mantissa bits, " + numLostBits + " bits lost in operation, and " + roundingUlpError + " ULP rounding error.")
 
-		/** @type {number} */ var refBits = tcuFloat.newFloat64(reference).bits();
```

Comment:
Calling `bits` on this value is invalid since 64 bits do not fit in a JS number.